nav2_amcl/src/amcl_node.cpp

```diff
@@ -266,6 +266,7 @@ AmclNode::on_activate(const rclcpp_lifecycle::State & /*state*/)
   // Lifecycle publishers must be explicitly activated
   pose_pub_->on_activate();
   particlecloud_pub_->on_activate();
+  particlecloudwithweights_pub_->on_activate();
```

Comment:
In the `on_configure` method can you do a RCLCPP_WARNING on the deprecation of the PoseArray topic?